agent/bench-scripts/postprocess/uperf-prepare-xmlfile

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+#echo "<profile name=\"protocol:\$protocol-test_type:\$test_type-wsize:\$wsize-threads:\$threads\">"
+
+echo "$@" >pre-script-args.txt
```

Comment:
Is this for debugging?  We only seem to look for `--test_type` and ignore the rest.

Reply:
Yes, it can be removed